packages/core/number/src/number.ts

```diff
@@ -6,4 +6,13 @@ function wrap(index: number, max: number) {
   return (max + index) % max;
 }
 
-export { clamp, wrap };
+// https://github.com/tmcw-up-for-adoption/simple-linear-scale/blob/master/index.js
+function linearScale(input: readonly [number, number], output: readonly [number, number]) {
```

Comment:
@benoitgrelard we spoke about moving things when they're used 3+ times but I had already done this before that discussion and it feels like its a bit too complex to be compying everywhere to me. Thoughts on leaving this here?

Reply:
`useStateMachine` is more types but hardly any code, it's not complex logic-wise. I guess it depends how we want to measure complexity... I don't see lines of code as a measurement in that regard personally but happy to duplicate if you feel strongly about it?

Reply:
Yeah, I'm not worried much about the code length itself, but I'm not sure complexity in general should be a huge factor in whether we abstract or not. I'd say there's just as much chance of something not getting updated in both places if duplicated whether it's complex or very simple, it's all just about remembering it which hopefully between us we would catch in code review. So I would still think let's just duplicate that one for now.